unikernel.ml

```diff
@@ -301,6 +304,39 @@ struct
               reply);
         []
 
+  let user_unikernel albatross ~user_name ~unikernel_name =
```

Comment:
the return value is `(Name.t * Unikernel.info) option * string option` -- where only ever exactly one option is set to `Some`.

We could use a `(Name.t * Unikernel.info, string) result` instead -- and the `Ok` and `Error ` constructors.